packages/cover-traffic-daemon/metrics.md

```diff
@@ -0,0 +1,53 @@
+On every tick of cover-traffic strategy, a few metrics are saved in the persisted state (`packages/cover-traffic-daemon/ct.json`).
+
+Other metrics can be deduced from logs. Note that logs start with `hopr:cover-traffic `
+
+1. At each tick, a log lists the current balance of the CT node, channels to open and to close.
+   ```sh
+   strategy tick: ${Date.now()} balance:${balance.toString()} open:${toOpen
```

Comment:
How does the output look like? I can imagine it by seeing the code but I would be easier to see some sample output